client/components/site-selector/index.jsx

```diff
@@ -246,6 +246,42 @@ export default React.createClass( {
 		);
 	},
 
+	renderStarredSites() {
+		const sites = this.props.sites.getStarred();
```

Comment:
IMHO it's a little misleading to name this `sites`, which I might assume elsewhere means "all my sites". Would rather call this `starredSites` and maybe the constant underneath as `starredSitesElements`?